app/src/main/java/com/android/kotlinlearning/Rastaurant1/RestaurantHome1.kt

```diff
@@ -0,0 +1,51 @@
+package com.android.kotlinlearning.Rastaurant1
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.android.kotlinlearning.R
+import com.android.kotlinlearning.databinding.FragmentRestraurant1Binding
+import kotlinx.android.synthetic.main.fragment_restraurant1.*
+
+class RestaurantHome1 : Fragment() {
+ private lateinit var binding: FragmentRestraurant1Binding
+ private val foodlist = ArrayList<Food>()
+ //val adapter:CastomAdapter1?=null
+
+ override fun onCreateView(
+ inflater: LayoutInflater,
+ container: ViewGroup?,
+ savedInstanceState: Bundle?
+
+
+ ): View? {
+
+ binding = FragmentRestraurant1Binding.inflate(inflater, container, false)
+ return binding.root
+ }
+
+ override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+ super.onViewCreated(view, savedInstanceState)
+ //binding.gridview.layoutManager = LinearLayoutManager(requireActivity())
+ foodlist.add(Food(  "Baboon"," Baboons are primates comprising the genus Papio, one of the 23 genera of Old World monkeys. There are six species of baboon: the hamadryas baboon, the Guinea baboon, the olive baboon, the yellow baboon, the Kinda baboon and the chacma baboon.",R.drawable.baboon))
```

Comment:
Check the list name, it contains animals name and list name is foodlist. And also while naming the list use camel case